Helper/Data.php

```diff
@@ -1024,4 +1019,27 @@ public function getAdyenMerchantAccount($method, $storeId)
         return $merchantAccount;
     }
 
+    public function formatTerminalAPIReceipt($paymentReceipt)
+    {
+        $paymentReceipt = json_decode($paymentReceipt);
+        $formatted = "<table class='terminal-api-receipt'>";
+        foreach ($paymentReceipt as $receipt) {
+            if ($receipt->DocumentQualifier == "CustomerReceipt") {
+                foreach ($receipt->OutputContent->OutputText as $item) {
+                    parse_str($item->Text, $get_array);
+                    $formatted .= "<tr class='terminal-api-receipt'>";
+                    if (!empty($get_array['name'])) {
+                        $formatted .= "<td class='terminal-api-receipt-name'>" . $get_array['name'] . "</td>";
```

Comment:
let's append the TDs even if they have empty content and ideally put &nbsp; http://jkorpela.fi/HTML/emptycells.html